client/suite.go

```diff
@@ -24,6 +24,8 @@ const (
 	retryConfigName          = "retry"
 	rpcTimeoutConfigName     = "rpc_timeout"
 	circuitBreakerConfigName = "circuit_break"
+	// 新增降级策略
```

Comment:
1. Avoid comments in Chinese
2. Comment with necessary and meaningful information, focusing more on `why`, not `what`